### ISSUE TITLE
Cannot finish the login flow

### ISSUE THREAD

**Issue by thiendangvan** [Portkey-Wallet/react-native-sdk#102]
**Describe the bug**
I cannot finish the login flow. After I verify my email -> create a PIN code -> verify my face ID -> I will be redirected to the create PIN code screen

**Necessary information**
package.json + yarn.lock files: https://drive.google.com/drive/folders/17xX2dR9_WNa56ws-Xvpa-zJmGO-M2q3p?usp=sharing

**To Reproduce**
1. verify email
2. create pin code
3. verify face ID

**Expected behavior**
should finish the login flow after verifying the face ID successfully

**Screenshots**
https://drive.google.com/file/d/1II82cnbw45zlZUqYHWM7Gx5MQioozEsw/view?usp=sharing

**Environment (please complete the following information):** <br>
 - OS: IOS
 - Version: 17.3.1
 - Portkey SDK version: 2.0.0-beta.2

**Comment by wade-portkey** [comment]
hi, @thiendangvan , we have fixed this bug yesterday, and the released version SDK will be released this evening, you can try it this evening or tomorrow morning.

**Comment by thiendangvan** [comment]
> hi, @thiendangvan , we have fixed this bug yesterday, and the released version SDK will be released this evening, you can try it this evening or tomorrow morning.

okay got it thanks thanks @wade-portkey . btw i wanna ask have you got this issue during development on Mac? The SDK shows a blank screen after calling the login method:
<image src="https://github.com/Portkey-Wallet/react-native-sdk/assets/154788705/7656e6fe-2605-4342-a6af-26e1e4d9695d" width="214" height="455" />

**Comment by wade-portkey** [comment]
> > hi, @thiendangvan , we have fixed this bug yesterday, and the released version SDK will be released this evening, you can try it this evening or tomorrow morning.
> 
> okay got it thanks thanks @wade-portkey . btw i wanna ask have you got this issue during development on Mac. The SDK shows a blank screen after calling the login method: <img alt="" width="214" height="455" src="https://private-user-images.githubusercontent.com/154788705/308096887-7656e6fe-2605-4342-a6af-26e1e4d9695d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwMjQyOTksIm5iZiI6MTcwOTAyMzk5OSwicGF0aCI6Ii8xNTQ3ODg3MDUvMzA4MDk2ODg3LTc2NTZlNmZlLTI2MDUtNDM0Mi1hNmFmLTI2ZTFlNGQ5Njk1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyN1QwODUzMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYjdhMWNiZjRkOTc5OTk1ZjgxM2UzZjkyMDJiNmNlNmRlZDdmY2U1YmFhMWNiYzgxMDE3NmI4NDA0ZmNkYjU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wCJf3LPylCTIHy9De5m9R_A5zhs-cG4fbR_YNRjvcfI">

We have not encountered this issue. I suspect it is caused by the same reason as the one above. I will release a new beta version that has fixed the above issue. You can try it, and if there are still problems, please tell us.

**Comment by wade-portkey** [comment]
> > > hi, @thiendangvan , we have fixed this bug yesterday, and the released version SDK will be released this evening, you can try it this evening or tomorrow morning.
> > 
> > 
> > okay got it thanks thanks @wade-portkey . btw i wanna ask have you got this issue during development on Mac. The SDK shows a blank screen after calling the login method: <img alt="" width="214" height="455" src="https://private-user-images.githubusercontent.com/154788705/308096887-7656e6fe-2605-4342-a6af-26e1e4d9695d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwMjQyOTksIm5iZiI6MTcwOTAyMzk5OSwicGF0aCI6Ii8xNTQ3ODg3MDUvMzA4MDk2ODg3LTc2NTZlNmZlLTI2MDUtNDM0Mi1hNmFmLTI2ZTFlNGQ5Njk1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyN1QwODUzMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYjdhMWNiZjRkOTc5OTk1ZjgxM2UzZjkyMDJiNmNlNmRlZDdmY2U1YmFhMWNiYzgxMDE3NmI4NDA0ZmNkYjU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wCJf3LPylCTIHy9De5m9R_A5zhs-cG4fbR_YNRjvcfI">
> 
> We have not encountered this issue. I suspect it is caused by the same reason as the one above. I will release a new beta version that has fixed the above issue. You can try it, and if there are still problems, please tell us.

The new beta version is [`2.0.0-beta.3`](https://www.npmjs.com/package/@portkey/react-native-sdk),  you can try it.

**Comment by thiendangvan** [comment]
> The new beta version is [`2.0.0-beta.3`](https://www.npmjs.com/package/@portkey/react-native-sdk), you can try it.

let me try

**Comment by thiendangvan** [comment]
> We have not encountered this issue. I suspect it is caused by the same reason as the one above. I will release a new beta version that has fixed the above issue. You can try it, and if there are still problems, please tell us.

hi @wade-portkey, I'm still getting the same issue after update the 2.0.0-beta.3.
steps:
1. update SDK version to 2.0.0-beta.3
2. npx expo prebuild --clean
3. expo ios
4. call the login method
-> show a blank screen

logs when I calling the login method:
```
DEV2FOD] [I] <MMKV.cpp:167::initialize> version v1.3.3, page size 16384, arch arm64-v8a
[DEV2FOD] [I] <MMKV.cpp:219::initializeMMKV> Apple Device: 0, version: 64, mlock enabled: 0
[DEV2FOD] [I] <MMKV.cpp:226::initializeMMKV> root dir:
/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv
[DEV2FOD] [I] <MMKV_OSX.cpp:99::setIsInBackground> g_isInBackground:0
[DEV2FOD] [I] <libMMKV.mm:116::+[MMKV initializeMMKV:logLevel:handler:]> appState:0
[DEV2FOD] [I] <MemoryFile.cpp:97::open> open fd[0x27],
/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk
[DEV2FOD] [I] <MemoryFile.cpp:202::mmap> mmap to address [0x1035a8000], oldPtr [0x0],
[/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk]
[DEV2FOD] [I] <MemoryFile_OSX.cpp:35::tryResetFileProtection> protection on
[/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk]
is (null)
[DEV2FOD] [I] <MemoryFile.cpp:97::open> open fd[0x28],
/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk.crc
[DEV2FOD] [I] <MemoryFile.cpp:202::mmap> mmap to address [0x1035cc000], oldPtr [0x0],
[/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk.crc]
[DEV2FOD] [I] <MemoryFile_OSX.cpp:35::tryResetFileProtection> protection on
[/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk.crc]
is (null)
[DEV2FOD] [I] <MMKV_IO.cpp:210::loadMetaInfoAndCheck> meta file [portkey-sdk] has flag [0]
[DEV2FOD] [I] <MMKV_IO.cpp:80::loadFromFile> loading [portkey-sdk] with 0 actual size, file size 16384,
InterProcess 0, meta info version:4
[DEV2FOD] [I] <MMKV_IO.cpp:129::loadFromFile> loaded [portkey-sdk] with 0 key-values
```

**Comment by thiendangvan** [comment]
if u need more info abt my environments like node version, pod version, macOS version, ... pls tell me

**Comment by wade-portkey** [comment]
> > We have not encountered this issue. I suspect it is caused by the same reason as the one above. I will release a new beta version that has fixed the above issue. You can try it, and if there are still problems, please tell us.
> 
> hi @wade-portkey, I'm still getting the same issue after update the 2.0.0-beta.3. steps:
> 
> 1. update SDK version to 2.0.0-beta.3
> 2. npx expo prebuild --clean
> 3. expo ios
> 4. call the login method
>    -> show a blank screen
> 
> logs when I calling the login method:
> 
> ```
> DEV2FOD] [I] <MMKV.cpp:167::initialize> version v1.3.3, page size 16384, arch arm64-v8a
> [DEV2FOD] [I] <MMKV.cpp:219::initializeMMKV> Apple Device: 0, version: 64, mlock enabled: 0
> [DEV2FOD] [I] <MMKV.cpp:226::initializeMMKV> root dir:
> /Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv
> [DEV2FOD] [I] <MMKV_OSX.cpp:99::setIsInBackground> g_isInBackground:0
> [DEV2FOD] [I] <libMMKV.mm:116::+[MMKV initializeMMKV:logLevel:handler:]> appState:0
> [DEV2FOD] [I] <MemoryFile.cpp:97::open> open fd[0x27],
> /Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk
> [DEV2FOD] [I] <MemoryFile.cpp:202::mmap> mmap to address [0x1035a8000], oldPtr [0x0],
> [/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk]
> [DEV2FOD] [I] <MemoryFile_OSX.cpp:35::tryResetFileProtection> protection on
> [/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk]
> is (null)
> [DEV2FOD] [I] <MemoryFile.cpp:97::open> open fd[0x28],
> /Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk.crc
> [DEV2FOD] [I] <MemoryFile.cpp:202::mmap> mmap to address [0x1035cc000], oldPtr [0x0],
> [/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk.crc]
> [DEV2FOD] [I] <MemoryFile_OSX.cpp:35::tryResetFileProtection> protection on
> [/Users/thiendang201/Library/Developer/CoreSimulator/Devices/CB00D97A-070A-45CB-808E-4225AB7B4724/data/Containers/Data/Application/FA4B5ADF-7848-4218-8678-E793DCA986C6/Documents/mmkv/portkey-sdk.crc]
> is (null)
> [DEV2FOD] [I] <MMKV_IO.cpp:210::loadMetaInfoAndCheck> meta file [portkey-sdk] has flag [0]
> [DEV2FOD] [I] <MMKV_IO.cpp:80::loadFromFile> loading [portkey-sdk] with 0 actual size, file size 16384,
> InterProcess 0, meta info version:4
> [DEV2FOD] [I] <MMKV_IO.cpp:129::loadFromFile> loaded [portkey-sdk] with 0 key-values
> ```

ok i know, i will attempt to reproduce this issue and resolve it.

**Comment by wade-portkey** [comment]
> > hi, @thiendangvan , we have fixed this bug yesterday, and the released version SDK will be released this evening, you can try it this evening or tomorrow morning.
> 
> okay got it thanks thanks @wade-portkey . btw i wanna ask have you got this issue during development on Mac? The SDK shows a blank screen after calling the login method: <img alt="" width="214" height="455" src="https://private-user-images.githubusercontent.com/154788705/308096887-7656e6fe-2605-4342-a6af-26e1e4d9695d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwMjQzNDMsIm5iZiI6MTcwOTAyNDA0MywicGF0aCI6Ii8xNTQ3ODg3MDUvMzA4MDk2ODg3LTc2NTZlNmZlLTI2MDUtNDM0Mi1hNmFmLTI2ZTFlNGQ5Njk1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyN1QwODU0MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YTliMzI2OGE1ZWU0Y2Q3ZTE2MWQ3ODkxNTRmMjNhODk4MTQ1ODc1NGQ0MmVhNGIyMTZlMzE4NmZkZGJiNzVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.t4BdqyuT8LYq216WY5rN1HdWaJeWv182N_KJrDBTGQI">

hi @thiendangvan , is this issue only present when screen recording? does this issue occur when not recording the screen? I want to confirm if this issue is related to screen recording.

**Comment by thiendangvan** [comment]
> hi @thiendangvan , is this issue only present when screen recording? does this issue occur when not recording the screen? I want to confirm if this issue is related to screen recording.

hi @wade-portkey, it is not related to screen recording. it happens on the IOS emulator only, android emulator work fine

**Comment by wade-portkey** [comment]
> > hi @thiendangvan , is this issue only present when screen recording? does this issue occur when not recording the screen? I want to confirm if this issue is related to screen recording.
> 
> hi @wade-portkey, it is not related to screen recording. it happens on the IOS emulator only, android emulator work fine

I didn't reproduce this issue here. Could you record the entire login flow so that I can try to find the reason.

**Comment by thiendangvan** [comment]
> > > hi @thiendangvan , is this issue only present when screen recording? does this issue occur when not recording the screen? I want to confirm if this issue is related to screen recording.
> > 
> > 
> > hi @wade-portkey, it is not related to screen recording. it happens on the IOS emulator only, android emulator work fine
> 
> I didn't reproduce this issue here. Could you record the entire login flow so that I can try to find the reason.

okay okay wait a sec

**Comment by thiendangvan** [comment]
hi @wade-portkey, pls take a look: https://drive.google.com/file/d/1v__DUvHtGY9n7PmEQmPcaUE2AaMcOV8p/view?usp=sharing

**Comment by thiendangvan** [comment]
@wade-portkey, may i ask if there are any updates for resolution in package.json?
current resolutions: 
```
"resolutions": {
    "@invertase/react-native-apple-authentication": "^2.2.2",
    "@portkey/react-native-sdk": "^2.0.0-beta.2",
    "@react-native-async-storage/async-storage": "^1.17.6",
    "@react-native-google-signin/google-signin": "^8.0.0",
    "expo": "~49.0.15",
    "expo-splash-screen": "~0.20.5",
    "expo-status-bar": "~1.6.0",
    "expo-system-ui": "~2.4.0",
    "expo-device": "~5.4.0",
    "expo-application": "~5.3.0",
    "expo-auth-session": "5.2.0",
    "expo-camera": "~13.4.4",
    "expo-web-browser": "~12.3.2",
    "expo-screen-capture": "~5.3.0",
    "lottie-react-native": "^5.1.5",
    "react": "~18.2.0",
    "react-native": "~0.72.4",
    "react-native-background-timer": "^2.4.1",
    "react-native-config": "^1.4.6",
    "react-native-fast-image": "^8.6.3",
    "react-native-gesture-handler": "~2.12.0",
    "react-native-get-random-values": "1.8.0",
    "react-native-localize": "^2.2.1",
    "react-native-mmkv": "^2.10.2",
    "react-native-pager-view": "6.2.0",
    "react-native-safe-area-context": "4.6.3",
    "react-native-spinkit": "1.5.1",
    "react-native-svg-uri": "^1.2.3",
    "react-native-vector-icons": "^9.1.0",
    "react-native-webview": "^13.6.0",
    "react-native-svg": "13.9.0"
  },
```

**Comment by thiendangvan** [comment]
> @wade-portkey, may i ask if there are any updates for resolution in package.json? current resolutions:
> 
> ```
> "resolutions": {
>     "@invertase/react-native-apple-authentication": "^2.2.2",
>     "@portkey/react-native-sdk": "^2.0.0-beta.2",
>     "@react-native-async-storage/async-storage": "^1.17.6",
>     "@react-native-google-signin/google-signin": "^8.0.0",
>     "expo": "~49.0.15",
>     "expo-splash-screen": "~0.20.5",
>     "expo-status-bar": "~1.6.0",
>     "expo-system-ui": "~2.4.0",
>     "expo-device": "~5.4.0",
>     "expo-application": "~5.3.0",
>     "expo-auth-session": "5.2.0",
>     "expo-camera": "~13.4.4",
>     "expo-web-browser": "~12.3.2",
>     "expo-screen-capture": "~5.3.0",
>     "lottie-react-native": "^5.1.5",
>     "react": "~18.2.0",
>     "react-native": "~0.72.4",
>     "react-native-background-timer": "^2.4.1",
>     "react-native-config": "^1.4.6",
>     "react-native-fast-image": "^8.6.3",
>     "react-native-gesture-handler": "~2.12.0",
>     "react-native-get-random-values": "1.8.0",
>     "react-native-localize": "^2.2.1",
>     "react-native-mmkv": "^2.10.2",
>     "react-native-pager-view": "6.2.0",
>     "react-native-safe-area-context": "4.6.3",
>     "react-native-spinkit": "1.5.1",
>     "react-native-svg-uri": "^1.2.3",
>     "react-native-vector-icons": "^9.1.0",
>     "react-native-webview": "^13.6.0",
>     "react-native-svg": "13.9.0"
>   },
> ```

I just updated to "@portkey/react-native-sdk": "^2.0.0-beta.3" and tried again but nothing change 😥

**Comment by wade-portkey** [comment]
> > hi, @thiendangvan , we have fixed this bug yesterday, and the released version SDK will be released this evening, you can try it this evening or tomorrow morning.
> 
> okay got it thanks thanks @wade-portkey . btw i wanna ask have you got this issue during development on Mac? The SDK shows a blank screen after calling the login method: <img alt="" width="214" height="455" src="https://private-user-images.githubusercontent.com/154788705/308096887-7656e6fe-2605-4342-a6af-26e1e4d9695d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwMzg2OTksIm5iZiI6MTcwOTAzODM5OSwicGF0aCI6Ii8xNTQ3ODg3MDUvMzA4MDk2ODg3LTc2NTZlNmZlLTI2MDUtNDM0Mi1hNmFmLTI2ZTFlNGQ5Njk1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyN1QxMjUzMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNzRjNDY1Y2Y4N2NkYTM5YjNjZjUzY2EyNWFkYmUyYzc1NWU3MGU2NmY4NzhkNjRiMWY2ZDVjMmVkNzQyMjNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.lT4LsITdNU0Ub4AGsLfLIY5QVms0hV5q00JvBslCjeo">

I can not reproduce this issue today. may i ask, is it showing the blank screen every time you call the login method now? before this, it was able to show the right content, right? Have you tried uninstalling and reinstalling the app?

**Comment by thiendangvan** [comment]
> I can not reproduce this issue today. may i ask, is it showing the blank screen every time you call the login method now? before this, it was able to show the right content, right? Have you tried uninstalling and reinstalling the app?
>
hi @wade-portkey, yes it shows a blank screen every time I call the login method
i think this issue is related to the expo-dev-client, u can see below video. before install the expo-dev-client i could see the login screen, but after i installed it i saw a blank screen:https://drive.google.com/file/d/1HVRMfrxURPyCqIMNlIMo6kTVq8OWsGL5/view?usp=sharing
steps:
1. npx expo install expo-dev-client
2. npx expo prebuild
3. yarn ios

**Comment by wade-portkey** [comment]
> > I can not reproduce this issue today. may i ask, is it showing the blank screen every time you call the login method now? before this, it was able to show the right content, right? Have you tried uninstalling and reinstalling the app?
> 
> hi @wade-portkey, yes it shows a blank screen every time I call the login method i think this issue is related to the expo-dev-client, u can see below video. before install the expo-dev-client i could see the login screen, but after i installed it i saw a blank screen:https://drive.google.com/file/d/1HVRMfrxURPyCqIMNlIMo6kTVq8OWsGL5/view?usp=sharing steps:
> 
> 1. npx expo install expo-dev-client
> 2. npx expo prebuild
> 3. yarn ios

hi, @thiendangvan . Sorry for this. It seems that there was an error in our development documentation that misled you. You don't need to use expo-dev-client. You can try removing expo-dev-client and then rerunning the project to see if it works correctly on Android and iOS.

**Comment by thiendangvan** [comment]
hi @wade-portkey, after removed the expo-dev-client it's work fine on both android & ios. but on ios the background is gray 
<img width="787" alt="Screenshot 2024-02-28 at 10 37 12" src="https://github.com/Portkey-Wallet/react-native-sdk/assets/154788705/d9ef6e35-4345-4547-aa16-a27d834b73d6">

**Comment by wade-portkey** [comment]
get it, i will find the reason of the issue and solve it.

**Comment by thiendangvan** [comment]
> get it, i will find the reason of the issue and solve it.

thanks thanks

**Comment by wade-portkey** [comment]
> hi @wade-portkey, after removed the expo-dev-client it's work fine on both android & ios. but on ios the background is gray <img alt="Screenshot 2024-02-28 at 10 37 12" width="787" src="https://private-user-images.githubusercontent.com/154788705/308409643-d9ef6e35-4345-4547-aa16-a27d834b73d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwOTE3NzksIm5iZiI6MTcwOTA5MTQ3OSwicGF0aCI6Ii8xNTQ3ODg3MDUvMzA4NDA5NjQzLWQ5ZWY2ZTM1LTQzNDUtNDU0Ny1hYTE2LWEyN2Q4MzRiNzNkNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyOFQwMzM3NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YTc4MDQ3ZjQxMThhYWNlMmNkOTY5YjAxMjY2NGQ3ZTJlNDRhM2ExYzAwNTRiNjM0Y2FmOTU2YjZiZjFkMmIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IhwNVtPUmPvQn3mvID5QqvWEMflPLBl1PicyaicyDZQ">

hi, @thiendangvan , the reason for this issue is that we didn’t upload the iOS image resource files when publishing the npm package. The problem has been fixed now, and the new version is 2.0.1, you can try it.

**Comment by thiendangvan** [comment]
> hi, @thiendangvan , the reason for this issue is that we didn’t upload the iOS image resource files when publishing the npm package. The problem has been fixed now, and the new version is 2.0.1, you can try it.

i see, let me try. thanks @wade-portkey

**Comment by thiendangvan** [comment]
thanks @wade-portkey. it's ok now. let me close this issue
